scripts/generate-usage-data/generate-usage-data.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import random
+import uuid
+import json
+
+import requests
+
+token = '<API Token>'
+EnterpriseCustomer = 'customer_id'
+StandardCustomer = 'customer_id'
+CustomerIds = [EnterpriseCustomer, StandardCustomer]
+userId = []
+
+for _ in range(58):
+    userId.append(uuid.uuid4().hex)
+
+def ingest(events):
+    remainder = events
+    with open('ingest.json', 'w') as f:
+        json.dump(events, f)
+    f.close()
+    print('events sent successfully')
+
+def random_monthly_target(n):
+    return random.randint(n, 2*n)
+
+events = []
+for cid in CustomerIds: 
+    customerId = cid
+    t = datetime.now(timezone.utc) - timedelta(days=33)
```

Comment:
why remove 33 days?

Reply:
make this configurable?

Reply:
You are only allowed to backfill data 34 days in the past...this basically send is backfilled usage data